release_notes/367-release-notes.adoc

```diff
@@ -26,14 +26,21 @@ Red Hat has improved the usability of {product-title} CI integrations. CI output
 
 [id="new-runtimeclass-policy-criteria"]
 === Runtime Class policy criteria
-Users can now use RHACS to define the container runtime configuration that may be used to run a pod's containers using the *Runtime Class* policy criteria.
+Users can now use {product-title-short} to define the container runtime configuration that may be used to run a pod's containers using the *Runtime Class* policy criteria.
 
 [id="important-bug-fixes"]
 == Important bug fixes
 
-* *ROX-7815*: Previously, when using RHACS with the Compliance Operator integration, RHACS did not respect or populate Compliance Operator `TailoredProfiles`. This has been fixed.
+* *ROX-7815*: Previously, when using {product-title-short} with the Compliance Operator integration, {product-title-short} did not respect or populate Compliance Operator `TailoredProfiles`. This has been fixed.
 * *ROX-7254*: Previously, the *Alpine Linux package manager (APK) in Image* policy looked for the presence of `apk` package in the image rather than the `apk-tools` package. This issue has been fixed.
 
+[id="resolved-in-version-3671"]
+=== Resolved in version 3.67.1
+
+Release date: December 6, 2021
+
+* *ROX-8698*: In {product-title-short} 3.67.0, the TLS verification would fail when you integrated {product-title-short} with {ocp} OAuth server for {ocp} 4.8 and newer. This has been fixed.
```

Comment:
```suggestion
* *ROX-8698*: In {product-title-short} 3.67.0, the TLS verification would fail when you integrated {product-title-short} with {ocp} OAuth server for {ocp} 4.8 and later. This issue has been fixed.
```
User `later` when referring to a subsequent software version or fix level. IBM style guide, (page358).